apps/survey/src/components/elements/TimePicker.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div>
+    <v-combobox
+      class="cb-time-picker"
+      hide-details
+      :items="timeItems"
+      label="Time"
+      outlined
+      :value="internalTime"
+      @change="updateTime"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, ref } from 'vue';
+
+export default defineComponent({
+  props: {
+    value: {
+      type: String,
+      required: true,
+    },
+    allowedMinutes: {
+      type: Number,
+      required: true,
+    },
+    is12HourFormat: {
```

Comment:
use same format as already defined in prompt props -> `am | pm`, so <component :is> can be used.